### DISCUSSION TITLE
Can not receive Illuminate\Notifications\Events\BroadcastNotificationCreated

### DISCUSSION THREAD

**Discussion by hamstag** [qruto/laravel-wave#39] [Q&A] [answered]
On php artisan queue:work it's two events BroadcastNotification and BroadcastNotificationCreated
<img width="1004" alt="Screen Shot 2023-08-31 at 08 57 04" src="https://github.com/qruto/laravel-wave/assets/99853768/df07a6c1-d850-4e57-a0ea-86f0d8cd86d6">

but on chrome browser only received BroadcastNotification
<img width="1162" alt="Screen Shot 2023-08-31 at 08 56 51" src="https://github.com/qruto/laravel-wave/assets/99853768/1765a388-4b43-4e05-ad67-321445437648">

**Comment by slavarazum** [comment]
`BroadcastNotificationCreated` is a system event, which uses to deliver events into your [notifications channel](https://laravel.com/docs/master/broadcasting#notifications).

**Reply by hamstag** [reply]
because when listening for notifications nothing happen.
```
Echo.private('App.Models.User.1')
    .notification((notification) => {
        console.log(notification.type);
    });
```

**Reply by hamstag** [reply]
Hi! the problem is from laravel/framework v10.21.0 they add broadcastAs function at BroadcastNotificationCreated https://github.com/laravel/framework/issues/48242

**Reply by slavarazum** [reply]
Thanks! Will process it soon.

**Reply by slavarazum** [reply, marked as answer]
Fixed in [v10.21.1](https://github.com/laravel/framework/commit/108f0156b0a0d8fea40f3392685f060e613e0829#diff-798bc51cae36da375dd3d55e421bffb4f7f798418a028352f35b1e063837cb9e).
